mqtt/mqtt-bridge/src/settings.rs

```diff
@@ -0,0 +1,202 @@
+use std::path::Path;
+use std::vec::Vec;
```

Comment:
If you use the rust-analyzer extension then you can merge these imports with the lightbulb button. It will show up if your cursor is over the duplicated usings. I can do a screenshare to show you how to configure the extension if you want.